packages/shorebird_cli/test/src/doctor_test.dart

```diff
@@ -6,11 +6,11 @@ import 'package:shorebird_cli/src/logger.dart';
 import 'package:shorebird_cli/src/validators/validators.dart';
 import 'package:test/test.dart';
 
-class _MockLogger extends Mock implements Logger {}
+class MockLogger extends Mock implements Logger {}
```

Comment:
Shouldn't these be imported from mocks as well?

Reply:
Done